src/BuildScriptGenerator/Node/NodeLanguageDetector.cs

```diff
@@ -149,5 +154,51 @@ private string DetectNodeVersion(dynamic packageJson)
 
             return nodeVersion;
         }
+
+        private bool IsStaticSite(ISourceRepo sourceRepo)
+        {
+            return IsHugoSite(sourceRepo);
+        }
+
+        private bool IsHugoSite(ISourceRepo sourceRepo)
+        {
+            // Hugo configuration variables: https://gohugo.io/getting-started/configuration/#all-configuration-settings
```

Comment:
Seems like there are 3 different formats for this: json, yaml and toml. We should handle all these?

Reply:
Added checks for each file.